go.mod

```diff
@@ -4,43 +4,52 @@ go 1.20
 
 replace k8s.io/client-go => k8s.io/client-go v0.20.0
 
+replace github.com/rancher/apiserver v0.0.0-20201023000256-1a0a904f9197 => github.com/KevinJoiner/rancher-apiserver v0.0.0-20240111144620-faf8426fbfac
```

Comment:
Remove and update apiserver commit after https://github.com/rancher/apiserver/pull/47 is merged